gui/default/index.html

```diff
@@ -285,7 +285,7 @@ <h4 class="panel-title">
                   <span ng-switch-when="idle"><span class="hidden-xs" translate>Up to Date</span><span class="visible-xs">&#9724;</span></span>
                   <span ng-switch-when="syncing">
                     <span class="hidden-xs" translate>Syncing</span>
-                    ({{syncPercentage(folder.id)}}%)
+                    ({{syncPercentage(folder.id)}}%, {{syncRemaining(folder.id)}})
```

Comment:
It seems that it might be unclear what the amount means. Size? Transferred? Remaining?